config/initializers/oracle.rb

```diff
@@ -1,4 +1,8 @@
 if Rails.env.development? || Rails.env.production?
+  if Rails.env.development?
```

Comment:
Why is it better to do it here?

Reply:
Before it was in the Gemfile, and rubocop complains about having stiff in there twice (in different groups). Also, before we had to add the `activerecord-oracle_enhanced-adapter` into the development group in order to run some rake commands. This way we no longer have to do that.